src/app/components/Timestamp/index.jsx

```diff
@@ -1,61 +1,27 @@
 import React from 'react';
 import styled from 'styled-components';
-import { number } from 'prop-types';
+import { string } from 'prop-types';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import { GEL_BREVIER } from '@bbc/gel-foundations/typography';
 import { C_RHINO } from '@bbc/psammead-styles/colours';
 import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
-import { T_BREVIER } from '../../lib/constants/typography';
-import { GEL_SPACING_DBL } from '../../lib/constants/styles';
-
-// if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
-const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
-
-const formatDateTime = dateObj => {
-  const fullYear = dateObj.getFullYear();
-  const monthTwoDigit = dateObj.toLocaleDateString('en-GB', {
-    month: '2-digit',
-  });
-  const dayTwoDigit = dateObj.toLocaleDateString('en-GB', {
-    day: '2-digit',
-  });
-
-  return `${fullYear}-${monthTwoDigit}-${dayTwoDigit}`;
-};
-
-const formatTimestamp = dateObj => {
-  const fullYear = dateObj.getFullYear();
-  const monthLong = dateObj.toLocaleDateString('en-GB', {
-    month: 'long',
-  });
-  const dayNumeric = dateObj.toLocaleDateString('en-GB', {
-    day: 'numeric',
-  });
-
-  return `${dayNumeric} ${monthLong} ${fullYear}`;
-};
 
 const StyledTimestamp = styled.div`
```

Comment:
can we not do this? `const StyledTimestamp = styled.time`

Cleaner than wrapping it in a styled `div` IMO

Reply:
Yes, the div was needed in a previous version of this component, where we had offscreen text being passed into this component as well. I will remove it.